store/cassandra/table.go

```diff
@@ -0,0 +1,76 @@
+package cassandra
+
+import (
+	"fmt"
+	"math"
+)
+
+const QueryFmtRead = "SELECT ts, data FROM %s WHERE key IN ? AND ts < ?"
+const QueryFmtWrite = "INSERT INTO %s (key, ts, data) values(?,?,?) USING TTL ?"
+
+// TTLTables stores table definitions keyed by their TTL
```

Comment:
am i right that the TTLs are in hours? if so i'd specify that, otherwise it's very easy to assume that's seconds